### ISSUE TITLE
[Bug]: The iOS client version 0.9.75-beta plays very slowly for the first time. But the web playback is very fast

### ISSUE THREAD

**Issue by thsrite** [advplyr/audiobookshelf#3283]
### What happened?

The iOS client version 0.9.75-beta plays very slowly for the first time. It takes about tens of seconds, and the more audiobooks there are, the slower the playback. I have tried both internal and external networks, as well as connecting to other people's servers.
The conclusion is that the iOS client plays very slowly, while the web or Android client plays very fast

### What did you expect to happen?

IOS client version 0.9.75-beta accelerates playback speed

### Steps to reproduce the issue

1. Open the iOS client
2. Find audiobooks with over 2000 episodes that have never been played before
3. Click on the streaming playback of audiobooks

### Audiobookshelf version

server：v2.12.3， iOS：0.9.75-beta

### How are you running audiobookshelf?

Docker

### What OS is your Audiobookshelf server hosted from?

Linux

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

None

### Additional Notes

https://github.com/user-attachments/assets/68e3ad05-ece5-485b-b1e3-88db60a65f5a

**Comment by thsrite** [comment]
The same audiobook will start playing within 3 seconds when it is first played on the web, but it will take about 40 seconds on the iOS client. The above video is the result of the iOS client's 0.9.7-beta test

**Comment by thsrite** [comment]
I test on these device， the same result
1. ipad pro  ios17.1.1    0.9.7-beta
2. iphone 12  ios16.6.1   0.9.7-beta
3. iphone x  ios16.7    0.9.7-beta

**Comment by nichwall** [comment]
Can you try app version 0.9.76?

**Comment by thsrite** [comment]
> Can you try app version 0.9.76?

I have already tried, the same problem

**Comment by nichwall** [comment]
What file format and what codec is the file? If it is not supported by direct play by the client (in this case iOS), then you will need to wait for the server to start transcoding the file. For example, opus requires transcoding on iOS but not Android or most web browsers.

**Comment by thsrite** [comment]
.m4a .mp3 AAC
---- Replied Message ----
FromNicholas ***@***.***>Date08/14/2024 21:06 ***@***.***> ***@***.***>,
***@***.***>SubjectRe: [advplyr/audiobookshelf] [Bug]: The iOS client version 0.9.75-beta plays very slowly for the first time. But the web playback is very fast (Issue #3283)
What file format and what codec is the file? If it is not supported by direct play by the client (in this case iOS), then you will need to wait for the server to start transcoding the file. For example, opus requires transcoding on iOS but not Android or most web browsers.

**Comment by advplyr** [comment]
The audiobook has 2000 audio files?

**Comment by advplyr** [comment]
I know that part of the problem will be the amount of data that Abs stores and sends for each audio file is way too much. That will be improved over time since we are working on that.
I doubt that is the full issue though. It may be that the iOS audio player is loading too much into memory.

**Comment by thsrite** [comment]
> I know that part of the problem will be the amount of data that Abs stores and sends for each audio file is way too much. That will be improved over time since we are working on that.
> I doubt that is the full issue though. It may be that the iOS audio player is loading too much into memory.

Yes, if there are too many files, the playback will stall, and there is a probability that the ios client will crash
